### ISSUE TITLE
Investigate how we do hg up

### ISSUE THREAD

**Issue by pascalchevrel** [mozfr/transvision#606]
We do "hg up -C", we should measure if the -C option is useful./has a time cost.

**Comment by flodolo** [comment]
Right now we [url=https://github.com/mozfr/transvision/blob/master/app/scripts/glossaire.sh#L124]do[/url] 
```
hg pull -r default
hg update -C
```

The alternative would be to just do
```
hg pull -u default
```
The rational for avoiding `-C` is that a Transvision installation doesn't really have any local changes to discard.

The gain in terms of time seems to be around 30%, and a lot higher for huge repositories like mozilla-central.

**Comment by flodolo** [comment]
I did another test on my local VM (running on a mechanical disk).

First I ran glossaire.sh once to update all repos (with `createTMX=false`), to minimize the differences between the two following runs.

Then I ran it again with `time`
```
real	35m50.462s
user	1m11.656s
sys	    1m14.296s
```

And then I ran the modified version from this changeset (still with `createTMX=false`)
https://github.com/flodolo/transvision/commit/2cde4f2e38c1637ac13aed0a0a75f0024af1633b
```
real	21m55.256s
user	0m37.384s
sys	    0m19.016s
```

The time for the 2nd run is slightly slower than real, since p12n_extract takes less time the 2nd time you run it on the same files. Still, it's an impressive difference.

Note: we actually need `hg pull -r default -u`, otherwise we'll pull a bunch of unnecessary branches.
